<launch>
<node pkg="rosserial_python" type="serial_node.py" name="serial">
  <param name="port" value="/dev/ttyACM2"/>
</node>

<node pkg="joyous_servo" type="joyous_servo_node" name="servo">
  <remap from="servo_angle" to="chatter" />
</node>

<node pkg="hokuyo_node" type="hokuyo_node" name="hokuyo">
  <param name="port" value="/dev/ttyACM1" />
  <param name="frame_id" value="servo" />
</node>

<node pkg="laser_converter" type="laser_converter_node" name="converter">
  <param name="output_frame" value="odom" />
  <remap from="scans_in" to="scan" />
  <remap from="points_out" to="sync_scan_cloud_filtered"/>
</node>

<node pkg="usb_cam" type="usb_cam_node" name="cam">
  <remap from="cam/image_raw" to="image/raw" />
</node>

</launch>
